README.md

```diff
@@ -36,36 +37,36 @@ npm i update-electron-app
 Drop this anywhere in your main process:
 
 ```js
-const { updateElectronApp } = require('update-electron-app')
-updateElectronApp()
+const { updateElectronApp } = require('update-electron-app');
+updateElectronApp();
```

Comment:
We don't include semicolons in examples in end user code, I'd prefer we didn't add these so that we remain consistent across our repos.

Reply:
[872445f](https://github.com/electron/update-electron-app/pull/172/commits/872445f54b646a50b9547c6c87319a74e6f9b742)

I think this used to be linted by `standard-markdown`. I've added `prettier-ignore` statements in accordance, but this is unlinted now. I can add the other dependency back in if needed.

Reply:
If we want Markdown linting in the future, for consistency we should use `@electron/lint-roller`.